tools/constraints/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
```

Comment:
Instead of adding this build setting, what are your thoughts on simply marking the test as `tags = ["manual"]`? Then we can add a separate CI step that only runs on Windows calling `bazel test //the/integration/test`. I think this is acceptable, given that we already have a CI step that installs WinFSP.

Reply:
Sure!

Reply:
That's slick!